doc/index.md

```diff
@@ -9,17 +9,20 @@ To build a new image, you have to:
 
 * be root
 * put the installation-images directory on a *local* file system (ie. not NFS)
+* have a valid .oscrc
 
-Then you can run ```make``` to build all parts.
+Then you must run ```make``` once to build all parts.
 
-The images are stored in the 'image' directory, the contents of the images
-is in the 'tmp' directory.
+The images will stored in the 'image' directory, the contents of the images
```

Comment:
Should this be "The images will be stored" instead of "The images will stored" ?